### DISCUSSION TITLE
What is the network policy for webhooks?

### DISCUSSION THREAD

**Discussion by lahwaacz** [longhorn/longhorn#9068] [Q&A]
[Documentation](https://longhorn.io/docs/1.6.2/references/networking/) says:

- Longhorn Manager
    - Ingress:
      ```
      Conversion Webhook Server	9501	TCP
      Admission Webhook Server	9502	TCP
      Recovery Backend Server   	9503	TCP
      ```

But the template `webhook-network-policy.yaml` sets these ports in the _ingress of the webhooks_:

https://github.com/longhorn/longhorn/blob/7e5b8047ed4acbd8c05264da6a721b4543f539de/chart/templates/network-policies/webhook-network-policy.yaml#L1-L33

So what is correct?

**Comment by lahwaacz** [comment]
I got confused by https://github.com/longhorn/longhorn/commit/7e5b8047ed4acbd8c05264da6a721b4543f539de which changed the label selectors just a few hours before I looked on the policy. Should have looked in the release tag rather than master...

**Reply by derekbit** [reply]
@lahwaacz Yes, master is for development purposes and the codes changes over time.
